src/app/features/balances-list/balances-list.tsx

```diff
@@ -54,8 +57,13 @@ export const BalancesList = ({ address, ...props }: BalancesListProps) => {
       {btcAssetBalance.balance.amount.isGreaterThan(0) && (
         <CryptoCurrencyAssetItem assetBalance={btcAssetBalance} icon={<Box as={BtcIcon} />} />
       )}
-      {stxAssetBalance.balance.amount.isGreaterThan(0) && (
-        <CryptoCurrencyAssetItem assetBalance={stxAssetBalance} icon={<StxAvatar {...props} />} />
+      {(stxAssetBalance.balance.amount.isGreaterThan(0) ||
+        stxUnachoredAssetBalance.balance.amount.isGreaterThan(0)) && (
```

Comment:
Why is this needed here now bc you removed subBalance?

Reply:
Unrelated, as previously still only the one value was checked.